doc/source/user_guide/AsynchronousAPI.ipynb

```diff
@@ -201,1629 +218,973 @@
      "text": [
-      "WARNING: InputWarning: Coordinate string is being interpreted as an ICRS coordinate provided in degrees. [astroquery.utils.commons]\n"
+      "- waiting for remote response (since 2021-08-31 17:57:36), please wait for https://www.astro.unige.ch/mmoda/dispatch-data/run_analysis\n",
+      "session: CHSJDFEI89QXQBX8 job: be2375ee6e04c3ef\n",
+      "\n",
+      "... query status \u001b[35mprepared\u001b[0m => \u001b[35mdone\u001b[0m\n",
+      "... assigned job id: \u001b[33mbe2375ee6e04c3ef\u001b[0m\n",
+      "\u001b[32mquery COMPLETED SUCCESSFULLY (state done)\u001b[0m\n",
+      "non-waiting dispatcher: terminating\n",
+      "- waiting for remote response (since 2021-08-31 17:57:51), please wait for https://www.astro.unige.ch/mmoda/dispatch-data/run_analysis\n"
      ]
     },
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "   SCW_ID   \n",
-      "------------\n",
-      "042000140010\n",
-      "042200020010\n",
-      "042200030010\n",
-      "042200040010\n",
-      "042200050010\n",
-      "042200060010\n",
-      "042200070010\n",
-      "042200080010\n",
-      "042200090010\n",
-      "042200100010\n",
-      "         ...\n",
-      "042201060010\n",
-      "042201070010\n",
-      "042201080010\n",
-      "042201090010\n",
-      "042201100010\n",
-      "042201110010\n",
-      "042201120010\n",
-      "042201120020\n",
-      "042201120030\n",
-      "042800690010\n",
-      "042800700010\n",
-      "Length = 114 rows\n",
-      "Is complete True\n",
-      "3 4 5\n",
-      "2006-07-01 00:00:00.0 - 2006-12-31 23:59:59.0\n"
+      "2005-01-01T00:00:00.0 - 2005-12-31T23:59:59.0\n"
      ]
     },
     {
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "WARNING: InputWarning: Coordinate string is being interpreted as an ICRS coordinate provided in degrees. [astroquery.utils.commons]\n"
+      "session: KITDUODRMMKQLUK9 job: 5ebc852ab66d5917\n",
+      "\n",
+      "... query status \u001b[35msubmitted\u001b[0m => \u001b[35mdone\u001b[0m\n",
+      "\u001b[32mquery COMPLETED SUCCESSFULLY (state done)\u001b[0m\n"
      ]
     },
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "   SCW_ID   \n",
-      "------------\n",
-      "046800040010\n",
-      "046800050010\n",
-      "048300410010\n",
-      "048300410020\n",
-      "048300410030\n",
-      "048300420010\n",
-      "048300420020\n",
-      "048300420030\n",
-      "048300430010\n",
-      "048300430020\n",
-      "         ...\n",
-      "048300580010\n",
-      "048300590010\n",
-      "048300600010\n",
-      "048300610010\n",
-      "048300620010\n",
-      "048300630010\n",
-      "048300640010\n",
-      "048300650010\n",
-      "048300660010\n",
-      "048300670010\n",
-      "048300680010\n",
-      "Length = 34 rows\n",
-      "Is complete True\n",
-      "3 4 5\n",
-      "complete 6 / 6\n",
+      "Is complete  True\n"
```

Comment:
it's pretty clear that this loop over multiple request args is very reusable and could be all in a function like `get_product_set()`. But users might want to implement some custom async requests too, so it's not bad to have this custom example. I would even keep this example and then add at the end "by the way, we made a convenience function for this". For the future.